src/server/pfs/server/api_server.go

```diff
@@ -452,11 +394,7 @@ func (a *apiServer) modifyFile(ctx context.Context, uw *fileset.UnorderedWriter,
 			}
 		case *pfs.ModifyFileRequest_CopyFile:
 			cf := mod.CopyFile
-			if err := func() (retErr error) {
-				func() { a.Log(ctx, cf, nil, nil, 0) }()
```

Comment:
do we still want to keep this logging since it's not at the RPC boundary?

Reply:
I debated this for a bit and decided not to keep it because none of the other branches in the switch log here.  I think if we want to log on this kind of thing, it should be done generically for all modify file operations.  Also, I suspect this may have just been left over from some debugging.